public/test/config.js

```diff
@@ -50,7 +50,7 @@ export default {
 		baseUrls: {
 			mediaBaseUrl: 'https://media.test.dev-gutools.co.uk',
 			apiBaseUrl: 'https://api.media.test.dev-gutools.co.uk',
-			videoBaseUrl: 'https://video.code.dev-gutools.co.uk',
-		},
+			videoBaseUrl: 'https://video.code.dev-gutools.co.uk'
+		}
     }
```

Comment:
the linter for the project was warning about trailing commas - this isn't actually a syntax concern anymore (other projects will be configured to require them) but thought it best to reduce the noise by compiling with the linting in place.